egs/wsj/s5/utils/segmentation.pl

```diff
@@ -353,7 +353,7 @@ sub print_segments {
     $i *= 10;
     $num_digits++;
   }
-  $format_str = "%0${num_digits}d"; # e.g. "%05d"
+  $format_str = "%0${num_digits}.0f"; # e.g. "%05.0f" round the same as seconds
```

Comment:
I don't understand the change.... Wouldn't it generate the same output as the previous one?
If you need correct rounding, you could also do 
$start_seconds * 100 + 0.5
or
int($start_seconds * 100 + 0.5)

Reply:
@jtrmal, Looks like perl does the rounding. So there will be rounding instead of truncating. But I do not know what exact consequences this change would cause.
```
$ perl -e 'printf "%03.0f %03.0f\n", 1.4999, 1.5'
001 002
$ perl -e 'printf "%03d %03d\n", 1.4999, 1.5'
001 001
```
Also, you've closed #3656 as resolved. So should this change go in or not?

> jtrmal commented on 20 Nov 2019
> I assume this is resolved.

Reply:
good question... I'd assume, since there wasn't any feedback, that the change is not necessary

Reply:
Thanks, I'll close it then.